internal/cmd/builtin/cmd/generate.go

```diff
@@ -180,16 +180,6 @@ func generate(cmd *cobra.Command, args []string) error {
 func generateFluxASTFile(dir string, pkg *ast.Package) error {
 	file := jen.NewFile(pkg.Package)
 	file.HeaderComment("// DO NOT EDIT: This file is autogenerated via the builtin command.")
-	file.Func().Id("init").Call().Block(
-		jen.Qual("github.com/influxdata/flux/runtime", "RegisterPackage").
```

Comment:
Note this is the only place where RegisterPackage was called that I know of. So it should be safe to remove that function implementation and clean up this process.